### PR TITLE
[DOC] Specify full form of LSS in beta series tutorial

### PR DESCRIPTION
**Pull request by TamerGezici**
Makes the text more coherent (as the full form for LSA is written above) and prevents people from scrolling up to remember what LSS stands for.

### CODE REVIEW

File: doc/changes/latest.rst [hunk 1]
```diff
@@ -30,6 +30,7 @@ Enhancements
 - :bdg-info:`Plotting` When plotting thresholded statistical maps with a colorbar, the threshold value(s) will now be displayed as tick labels on the colorbar (:gh:`#2833` by `Nicolas Gensollen`_).
 - :bdg-success:`API` :class:`~maskers.NiftiSpheresMasker` now has ``generate_report`` method (:gh:`3102` by `Yasmin Mzayek`_ and `Nicolas Gensollen`_).
 - :bdg-primary:`Doc`  Mention the classification type (all-vs-one) in  :ref:`sphx_glr_auto_examples_02_decoding_plot_haxby_glm_decoding.py` (:gh:`4122` by `Tamer Gezici`_). 
+- :bdg-primary:`Doc`  Specify full form of LSS in  :ref:`plot_beta_series.py` (:gh:`4122` by `Tamer Gezici`_).
```
Comment (ymzayek) [inline doc/changes/latest.rst:33]:
```suggestion
- :bdg-primary:`Doc`  Specify full form of LSS in  :ref:`sphx_glr_auto_examples_07_advanced_plot_beta_series.py` (:gh:`4141` by `Tamer Gezici`_). 
```

File: examples/07_advanced/plot_beta_series.py [hunk 2]
```diff
@@ -162,7 +162,7 @@
 # %%
 # Define the LSS models
 # ---------------------
-# We will now create a separate LSS model for each trial of interest.
+# We will now create a separate least squares seperate (LSS) model for each trial of interest.
```
Comment (ymzayek) [inline examples/07_advanced/plot_beta_series.py:165]:
```suggestion
# We will now create a separate Least Squares- Separate (LSS) model for each trial
# of interest.
```

**Review by bthirion — APPROVED** [review]
LGTM, thx.

File: examples/07_advanced/plot_beta_series.py [hunk 3]
```diff
@@ -162,7 +162,8 @@
 # %%
 # Define the LSS models
 # ---------------------
-# We will now create a separate LSS model for each trial of interest.
+# We will now create a separate Least Squares- Separate (LSS) model for each trial
+# of interest.
```
Comment (ymzayek) [inline examples/07_advanced/plot_beta_series.py:166]:
```suggestion
# We will now create a separate Least Squares- Separate (LSS) model for each
# trial of interest.
```

**Review by ymzayek — APPROVED** [review]
thanks @TamerGezici this one should be good to go once the tests pass

**Pull request merged**